tools/depends/target/Makefile

```diff
@@ -81,6 +81,10 @@ else ifeq ($(TARGET_PLATFORM),aml)
   DEPENDS += libamcodec
 endif
 
+ifeq ($(TARGET_PLATFORM),$(filter $(TARGET_PLATFORM),raspberry-pi aml gbm))
```

Comment:
`TARGET_PLATFORM` can only have the following values: appletvos, iphoneos, aml, gbm, mir or wayland
For android, linux with X11 and osx it is empty and the ifeq is therefore also true. At least for android and osx we don't want to build those libs.

Reply:
@Rechi any ideas what I can do here instead? I'm not that good with Makefile syntax

Reply:
```
ifeq ($(TARGET_PLATFORM),)
else ifeq ($(TARGET_PLATFORM),$(filter $(TARGET_PLATFORM),raspberry-pi aml gbm))
  DEPENDS += libxkbcommon libinput libudev libevdev mtdev
endif
```